src/Surfnet/ServiceProviderDashboard/Application/Service/ChangeRequestServiceInterface.php

```diff
@@ -19,8 +19,9 @@
 namespace Surfnet\ServiceProviderDashboard\Application\Service;
 
 use Surfnet\ServiceProviderDashboard\Application\Dto\ChangeRequestDtoCollection;
+use Surfnet\ServiceProviderDashboard\Domain\Entity\Entity\Protocol;
 
 interface ChangeRequestServiceInterface
 {
-    public function findById(string $id): ChangeRequestDtoCollection;
+    public function findByIdAndProtocol(string $id, Protocol $protocol): ChangeRequestDtoCollection;
```

Comment:
Great name change!

I sometimes also like to keep the base name of the method short, and let the parameters describe by what criteria we are searching. Example:

`findBy(string $id, Protocol $protocol)`

But please leave it as-is!